src/components/containers/TraceAccordion.js

```diff
@@ -24,14 +24,36 @@ class TraceAccordion extends Component {
     // we don't want to include analysis transforms when we're in the create panel
     const base = props.canGroup ? context.fullData : context.data;
 
-    this.filteredTracesFullDataPositions = [];
-    this.filteredTraces = base.filter((t, i) => {
-      if (props.excludeFits) {
-        return !(t.transforms && t.transforms.every(tr => tr.type === 'fit'));
+    this.filteredTracesFullDataPositions = base.map((t, i) => {
+      if (
+        props.excludeFits &&
+        !(t.transforms && t.transforms.every(tr => tr.type === 'fit'))
+      ) {
+        return i;
+      } else if (
+        props.excludeNonTransformable &&
+        TRANSFORMABLE_TRACES.includes(t.type)
+      ) {
+        return i;
+      } else if (
+        props.excludeNoColorbar &&
+        ((context.fullData[i].marker &&
```

Comment:
this code is repeated in the PanelMenu piece that hides the ColorBar panel... can you think of a way to centralize it?